rakelib/plugins-metadata.json

```diff
@@ -0,0 +1,905 @@
+{
+  "logstash-codec-cloudtrail": {
+    "default-plugins": false,
```

Comment:
is there particular reason for choosing to list by plugins instead of by types? for example:
```
{
  "default-plugins" : [
    "logstash-codec-dots",
    "logstash-codec-collectd",
    ...
  ],
  "core-specs": [
    ...
  ],
  ...
}
```